schemas/WebpackOptions.json

```diff
@@ -930,7 +930,19 @@
         },
         "sideEffects": {
           "description": "Flags a module as with or without side effects",
-          "type": "boolean"
+          "anyOf": [
+            {
+              "type": "boolean"
+            },
+            {
+              "items": {
+                "description": "Globs for child modules with side effects",
+                "minLength": 1,
+                "type": "string"
```

Comment:
What about regexps? Since ou are converting globs into regexps, supporting them would be trivial.

Reply:
I can add that. There was also the suggestion to support an object, but I wasn’t sure how far to go. I’ll add support for more variations.

Reply:
7d8700f adds support for all the variants I've run across in the issues.  I need help defining these in the schema.